lib/node_modules/@stdlib/utils/some-in-by/README.md

```diff
@@ -0,0 +1,234 @@
+}
+
+var obj = {
+    'a': 1,
+    'b': -2,
+    'c': 3,
+    'd': -1
+};
+
+var bool = someInBy( obj, 2, isNegative );
+// returns true
+```
+
+Once the function finds `n` successful properties, the function **immediately** returns `true`.
+
+```javascript
+function isPositive( value ) {
+    if ( value < 0 ) {
+        throw new Error( 'should never reach this line' );
+    }
+    return ( value > 0 );
+}
+
+var obj = {
+    'a': 1,
+    'b': 2,
+    'c': -3,
+    'd': 4
+};
+
+var bool = someInBy( obj, 2, isPositive );
+// returns true
+```
+
+The invoked `function` is provided three arguments:
+
+-   `value`: object property value
+-   `key`: object property key
+-   `obj`: input object
+
+To set the function execution context, provide a `thisArg`.
+
+```javascript
+function sum( value ) {
+    this.sum += value;
+    this.count += 1;
+    return ( value < 0 );
+}
+
+var obj = {
+    'a': 1,
+    'b': 2,
+    'c': 3,
+    'd': -5
+};
+
+var context = {
+    'sum': 0,
+    'count': 0
+};
+
+var bool = someInBy( obj, 1, sum, context );
+// returns true
+
+var mean = context.sum / context.count;
+// returns 0.25
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If provided an empty `obj`, the function returns `false`.
+
+    ```javascript
+    function alwaysTrue() {
+        return true;
+    }
+    var bool = someInBy( {}, 1, alwaysTrue );
+    // returns false
+    ```
+
+-   The function does **not** skip `undefined` properties.
+
+    ```javascript
+    function log( value, key ) {
+        console.log( '%s: %s', key, value );
+        return ( value < 0 );
+    }
+
+    var obj = {
+        'a': 1,
+        'b': void 0,
+        'c': void 0,
+        'd': 4,
+        'e': -1
+    };
+
+    var bool = someInBy( obj, 1, log );
+    // logs
+    // a: 1
+    // b: void 0
+    // c: void 0
+    // d: 4
+    // e: -1
+    ```
+
+-   The function provides limited support for dynamic objects (i.e., objects whose properties change during execution).
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+```javascript
+var randu = require( '@stdlib/random/base/randu' );
+var someInBy = require( '@stdlib/utils/some-in-by' );
+
+function threshold( value ) {
+    return ( value > 0.95 );
+}
+
+var bool;
+var obj = {};
+var i;
+
+for ( i = 0; i < 100; i++ ) {
+    obj[ 'key' + i ] = randu();
+}
+
+bool = someInBy( obj, 5, threshold );
+// returns <boolean>
+```
+
+</section>
+
+<!-- /.examples -->
+
+<section class="references">
+
+</section>
+
+<!-- /.references -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/utils/any-by`][@stdlib/utils/any-by]</span><span class="delimiter">: </span><span class="description">test whether at least one element in a collection passes a test implemented by a predicate function.</span>
```

Comment:
@performant23 No reason to fill in these; the related packages sections get auto-populated.